response_operations_ui/controllers/message_controllers.py

```diff
@@ -1,8 +1,8 @@
 import logging
 
-import jwt
 import requests
-from flask import current_app
+from flask import current_app, session
```

Comment:
alphabetised imports

Reply:
Ok - we should probably agree a standard so we can get it in sync across the teams linters

Reply:
Agreed